test/test_utils/security.py

```diff
@@ -494,8 +494,11 @@ def conduct_failure_routine(
         "fixable_vulnerabilities": fixable_list,
         "non_fixable_vulnerabilities": newly_found_non_fixable_list,
     }
-    ## TODO: Remove the is_pr_context before merging ##
-    if test_utils.is_canary_context() and test_utils.is_time_for_invoking_ecr_scan_failure_routine_lambda():
+    ## TODO: Make the conditions below as if test_utils.is_canary_context() and test_utils.is_time_for_invoking_ecr_scan_failure_routine_lambda() and os.getenv("REGION") == test_utils.DEFAULT_REGION:
+    ## to make sure that we just invoke the ECR_SCAN_FAILURE_ROUTINE_LAMBDA once everyday
+    if test_utils.is_canary_context() and os.getenv("REGION") == test_utils.DEFAULT_REGION:
```

Comment:
I am planning to push it like this to get feedback faster from the canaries. Once the canaries succeed, I will revert this as per the TODO statement I have commented above.

Reply:
understood - thanks for explanation